src/utils.ts

```diff
@@ -400,7 +399,7 @@ function getScriptTypeName(type: string) {
  * Handles error of each command.
  */
 // tslint:disable-next-line:no-any
-export function handleError(command: (...args: any[]) => Promise<void>) {
+export function handleError(command: (...args: any[]) => Promise<any>) {
```

Comment:
`void` is better than `any`. I think this returns `void` and should be kept this way.

Reply:
@grant There are three instances of `handleError` usage which fails when `Promise<void>` is expected.

```ts
  .action(handleError(openCmd));
```

```ts
  .action(handleError(logs));
```

```ts
  .action(handleError(apis));
```

Those three functions return `Promise<ChildProcess | undefined>`

To fix those linting errors, we can either redefine `handleError` to accept either `(...args: any[]) => Promise<any>` or `(...args: any[]) => Promise<unknown>`

Else `openCmd`, 'logs' and `apis` require to be adjusted somehow. Since I am not familiar with these parts of Clasp, I did not invest too much in the latest option.

Please advise.